roofit/hs3/test/histfactory.py

```diff
@@ -81,18 +81,18 @@ def test_closure(self):
 
         pdf.fitTo(
             data,
-            Strategy=1,
-            Minos=mc.GetParametersOfInterest(),
+            Strategy = 1,
+            Minos = mc.GetParametersOfInterest(),
             GlobalObservables = mc.GetGlobalObservables()
         )
-
+        
```

Comment:
There are some formatting errors in this file, e.g. the trailing whitespaces here. In RooFit, we are formatting out Python source files with [black](https://github.com/psf/black) and a character limit of 120. Can you please iron over this file with `black --line-length=120 histfactory.py`?